src/Elmish.WPF/ViewModels.fs

```diff
@@ -223,3 +227,102 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
     member _.ErrorsChanged = (helper :> INotifyDataErrorInfo).ErrorsChanged
     member _.HasErrors = (helper :> INotifyDataErrorInfo).HasErrors
     member _.GetErrors name = (helper :> INotifyDataErrorInfo).GetErrors name
+
+open System.Runtime.CompilerServices
+
+type [<AllowNullLiteral>] ViewModelBase<'model, 'msg>(args: ViewModelArgs<'model, 'msg>)
+  as this =
+
+  let mutable setBindings = Map.empty<String, VmBinding<'model, 'msg, obj>>
+
+  let mutable helper = ViewModelHelper.empty (fun () -> this) args
+
+  let { loggingArgs = loggingArgs
+        initialModel = initialModel
+        dispatch = dispatch } = args
+  let { log = log; nameChain = nameChain } = loggingArgs
+
+  let initializeBinding initializedBindings binding =
+    Initialize(loggingArgs, binding.Name, ViewModelHelper.getFunctionsForSubModelSelectedItem loggingArgs initializedBindings)
+      .Recursive(initialModel, dispatch, (fun () -> this |> IViewModel.currentModel), binding.Data)
+
+  member _.Get<'a> ([<CallerMemberName>] ?memberName: string) =
+    fun (binding: string -> Binding<'model, 'msg>) ->
+      let result =
+        option {
+          let! name = memberName
+          let! vmBinding = option {
+            match helper.Bindings.TryGetValue name with
+            | true, value ->
+              return value
+            | false, _ ->
+              let binding = binding name
+              let! vmBinding = initializeBinding helper.Bindings binding
+              let newBindings = helper.Bindings.Add (name, vmBinding)
+              let newValidationErrors =
+                FirstValidationErrors().Recursive(vmBinding)
+                |> Option.map (fun errorList -> helper.ValidationErrors.Add (name, errorList))
+                |> Option.defaultValue helper.ValidationErrors
+              helper <-
+                { helper with
+                    Bindings = newBindings
+                    ValidationErrors = newValidationErrors }
+              return vmBinding
+          }
+          return Get(nameChain).Recursive(helper.Model, vmBinding)
+        }
+      match result with
+      | None ->
+        log.LogError("[{BindingNameChain}] Get FAILED: Binding {BindingName} could not be constructed", nameChain, memberName)
+        failwithf "[%s] Get FAILED: Binding %O could not be constructed" nameChain memberName
```

Comment:
```suggestion
        failwithf $"[%s{nameChain}] Get FAILED: Binding {memberName} could not be constructed"
```

Reply:
Was this fixed already in the other part that has this code?

Reply:
I'm not seeing this code anywhere else.

Reply:
No, it's not.

Reply:
I see no other use cases not using `LogError` string templates.